src/Tests/Microsoft.DotNet.GenAPI.Tests/CSharpFileBuilderTests.cs

```diff
@@ -1389,5 +1389,163 @@ public partial class foo
                     """,
                 includeInternalSymbols: false);
         }
+
+        [Fact]
+        public void TestInternalDefaultConstructorGeneration()
+        {
+            RunTest(original: """
+                    namespace A
+                    {
+                        public class Bar
+                        {
+                            public Bar(int a) { }
+                        }
+
+                        public class Foo : Bar
+                        {
+                            internal Foo() : base(1) { }
+                        }
+                    }
+                    namespace B
+                    {
+                        public class Bar
+                        {
+                            public Bar(int a) { }
+                        }
+                    
+                        public class Foo : Bar
+                        {
+                            private Foo() : base(1) { }
+                        }
+                    }
+                    """,
+                expected: """
+                    namespace A
+                    {
+                        public partial class Bar
+                        {
+                            public Bar(int a) { }
+                        }
+
+                        public partial class Foo : Bar
+                        {
+                            internal Foo() : base(default) { }
```

Comment:
Maybe a naive question but why don't we provide 1 as the parameter and instead use `default`?

Reply:
I don't think we have access to parameter of the base class constructor call as this is part of implementation details. `internal Foo() : base(1) { }`. Technically, this doesn't meter and `default` is universal solution.